docs/index.md

```diff
@@ -17,6 +17,7 @@ maxdepth: 1
 Getting Started <getting_started.md>
 File Formats <file_formats.md>
 Command Line Interface <cli.rst>
+Casanovo Nextflow Workflow <nextflow.md>
```

Comment:
**thought:** Not sure whether the "Casanovo" prefix is needed, because it could be used for almost all pages then.